spec/controllers/api/v2/geographical_areas_controller_spec.rb

```diff
@@ -2,61 +2,65 @@
   before do
     Rails.cache.clear
     allow(Rails.cache).to receive(:fetch).and_call_original
+
+    country_geographical_area
+    group_geographical_area
+    region_geographical_area
+    globally_excluded_area
   end
 
-  let!(:country_geographical_area) { create(:geographical_area, :with_description, :country) }
-  let!(:group_geographical_area) { create(:geographical_area, :with_description, :group) }
-  let!(:region_geographical_area) { create(:geographical_area, :with_description, :region) }
+  let(:country_geographical_area) { create(:geographical_area, :with_description, :country) }
+  let(:group_geographical_area) { create(:geographical_area, :with_description, :group) }
+  let(:region_geographical_area) { create(:geographical_area, :with_description, :region) }
+  let(:globally_excluded_area) { create(:geographical_area, :with_description, :globally_excluded) }
 
   let(:actual_date) { Time.zone.today }
 
   describe 'GET countries' do
     subject(:do_response) { get :countries }
 
-    let(:pattern) do
-      {
-        data: [
-          { id: String, type: String, attributes: { id: String, description: String, geographical_area_id: String }, relationships: { children_geographical_areas: { data: [] } } },
-          { id: String, type: String, attributes: { id: String, description: String, geographical_area_id: String }, relationships: { children_geographical_areas: { data: [] } } },
-        ],
-        included: Array,
-      }
-    end
-
-    it { expect(do_response.body).to match_json_expression(pattern) }
-
     it { expect(do_response.body).to include(region_geographical_area.geographical_area_id) }
+    it { expect(do_response.body).to include(country_geographical_area.geographical_area_id) }
+    it { expect(do_response.body).not_to include(group_geographical_area.geographical_area_id) }
+    it { expect(do_response.body).not_to include(globally_excluded_area.geographical_area_id) }
 
     it 'caches the serialized countries' do
       do_response
 
-      expect(Rails.cache).to have_received(:fetch).with("_geographical-areas-countries-#{actual_date}", expires_in: 24.hours)
+      expect(Rails.cache).to have_received(:fetch).with("_geographical-areas-#{actual_date}-true-false", expires_in: 24.hours)
+    end
+
+    context 'when the exclude none filter is passed' do
+      subject(:do_response) { get :countries, params: { filter: { exclude_none: 'true' } } }
```

Comment:
have you tried with 
`params: { filter: { exclude_none: 'false' } }`

and should we consider capitalized version FALSE/TRUE? (what happens in that cases?)

Reply:
I could do that but the reality is I'm never going to use the api in that way :thinking: and this is me driving it.

I'm also never going to document this feature as this is an endpoint used by the frontend, only